xml/admin_ceph_gateway.xml

```diff
@@ -3047,115 +2977,89 @@ us-east-1.rgw.meta</screen>
-    <para>
-     You need to mention the &ogw; zone and port options in the configuration
-     file before starting the &ogw;. For more information on &ogw; and its
-     configuration, see <xref linkend="cha-ceph-gw"/>. The configuration
-     section of &ogw; should look similar to this:
+    <title>Start the Gateway</title>
+    <para>On the &ogw; host, start and enable the &cogw; service:
     </para>
-<screen>[client.rgw.us-east-1]
-rgw_frontends="beast port=80"
-rgw_zone=us-east-1</screen>
-    <para>
-     Start the &ogw;:
-    </para>
-<screen>&prompt.root;systemctl start ceph-radosgw@rgw.us-east-1</screen>
+<screen>&prompt.cephuser.ogw;systemctl start ceph-radosgw@rgw.`hostname -s`
+&prompt.cephuser.ogw;systemctl enable ceph-radosgw@rgw.`hostname -s`</screen>
    </sect3>
   </sect2>
 
-  <sect2 xml:id="ceph-rgw-fed-secondaryzone">
-   <title>Creating a Secondary Zone</title>
-   <para>
-    Zones within a zonegroup replicate all data to ensure that each zone has
-    the same data. When creating the secondary zone, execute all of the
-    following operations on a host identified to serve the secondary zone.
+  <sect2 xml:id="ceph-rgw-config-secondaryzone">
+   <title>Configure Secondary Zones</title>
+   <para>Zones within a &zonegroup; replicate all data to ensure that each
+     zone has the same data. When creating the secondary zone, execute
+     all of the following operations on a host identified to serve the secondary zone.
    </para>
-   <para>
-    In the secondary cluster, create and configure the secondary zone named
-    <literal>us-east-2</literal>. You can execute all the following commands in
-    the node hosting the master zone itself.
-   </para>
-   <para>
-    To create the secondary zone, use the same command as when you created the
-    primary zone, except dropping the master flag:
+   <note>
+     <para>To add a third zone, follow the same procedures as for adding
+       the secondary zone. Use different zone name.</para>
+   </note>
+   <important>
+     <para>You must execute metadata operations, such as user creation,
+       on a host within the master zone. The master zone and the secondary
+       zone can receive bucket operations, but the secondary zone redirects
+       bucket operations to the master zone. If the master zone is down,
+       bucket operations will fail.</para>
+   </important>
+
+   <sect3 xml:id="ceph-rgw-pull-realm">
+     <title>Pull The Realm</title>
+     <para>Using the URL path, access key, and secret of the master zone
+       in the master &zonegroup;, pull the realm configuration to the
+       host. To pull a non-default realm, specify the realm
+       using the <option>--rgw-realm</option> or <option>--realm-id</option>
+       configuration options.</para>
+     <note>
+       <para>Pulling the realm also retrieves the remote’s current period
+         configuration, and makes it the current period on this host as well.</para>
+     </note>
```

Comment:
I think we forgot to add the command to pull the realm from the master.

Reply:
Good catch.